Dockerfile

```diff
@@ -1,7 +1,18 @@
+FROM --platform=linux/amd64 mambaorg/micromamba:latest as micromamba
+# First Stage of the build, gets the jar for encyclopedia
+FROM --platform=linux/amd64 openjdk:8-jre
 ARG VERSION=1.12.34
```

Comment:
**Question:** We need to bump this to 2.x right?

Reply:
I cannot believe I did not change this ...

Reply:
changed in cce4906363f095402c8795da5f3205ead75652e9